### PR TITLE
feature/roadmap-goals

### PR DESCRIPTION
**Pull request by dannystey**
created roadmap & goals page for issue #50 #51 and #52 
- added a new roadmap-component and implemented a new page 'Roadmap & Goals'

i wrote the roadmap with my own words ... maybe sombody could review the content carefully

### CODE REVIEW

File: src/roadmap-goals.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+The first part of your journey is to get an overview of the topics, that come across during your first weeks. You will learn which vision the onboarding process follows and get in touch with our toolset like JIRA and GIT/BITBUCKET
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Fundamental I
+In that first part of the fundamentals of frontend development, you will be shown how to handle the mobile first approach, html semantics and css. Beside that you also get knowledge about accessability with WAI-ARIA and at least how to handle the SVG Icons.
+
+__Challanges__
```
Comment (SheepFromHeaven) [inline src/roadmap-goals.md]: Typo: ChallEnges

File: src/roadmap-goals.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+### Getting started
+
+The first part of your journey is to get an overview of the topics, that come across during your first weeks. You will learn which vision the onboarding process follows and get in touch with our toolset like JIRA and GIT/BITBUCKET
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Fundamental I
```
Comment (SheepFromHeaven) [inline src/roadmap-goals.md]: We have FundamentalS everywhere, so we should do it here too I guess

File: src/roadmap-goals.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+- Responsive Webdesign
+- SVG Icons
+
+*Goal: you know how to create a modern static web page*
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Fundamental II
```
Comment (SheepFromHeaven) [inline src/roadmap-goals.md]: We have FundamentalS everywhere, so we should do it here too I guess

File: src/roadmap-goals.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+The second part of fundamentals of frontend development will take you into the css preprocessor world with SASS / SCSS and also bring you to frontend developers magic potion: JavaScript
+
+__Challanges__
```
Comment (SheepFromHeaven) [inline src/roadmap-goals.md]: Typo: ChallEnges

File: src/roadmap-goals.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+*Goal: you understand why we value components as building blocks of our frontends and you are able to implement interactive and animated components that can communicate with each other*
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Biotope - Basic Component
+Biotope is the VI default Frontend Ecosystem, which is implemented in every project of VI. You will be shown how biotope works and which architecture relies to it and how you can build simple components.
+
+__Challanges__
```
Comment (SheepFromHeaven) [inline src/roadmap-goals.md]: Typo: ChallEnges

File: src/roadmap-goals.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+*Goal: you understand how Biotope helps us to build scalable and reusable frontend frameworks and you are able to implement a simple component in a typical Biotope project*
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Biotope - Advanced Component
+You will dive deeper into the functionalities of webcomponents and also typescript ( a typebased javascript language ). You will also be shown how to handle APIs correctly with the fetch API.
+
+__Challanges__
```
Comment (SheepFromHeaven) [inline src/roadmap-goals.md]: Typo: ChallEnges

File: src/roadmap-goals.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+- Extend the accordion with typescript and API requests
+
+*Goal: you know advanced coding techniques to build more complex components and how to improve code quality through types and unit tests*
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Testing
+In that topic you'll dive deeper into unit testing via diffrent tools and learn more about the TDD approach.
```
Comment (SheepFromHeaven) [inline src/roadmap-goals.md]: Typo: DiffErent

File: src/roadmap-goals.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+- Build some unit tests with jest, mocha and chai
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Biotope Ecosystem
+You will be shown the whole ecosystem of biotope and afterwards you be able to work on biotope and bring it to the next levels.
+
+__Challanges__
```
Comment (SheepFromHeaven) [inline src/roadmap-goals.md]: Typo: ChallEnges

File: src/roadmap-goals.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+*Goal: you know advanced coding techniques to build more complex components and how to improve code quality through types and unit tests*
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Testing
+In that topic you'll dive deeper into unit testing via diffrent tools and learn more about the TDD approach.
+
+__Challanges__
```
Comment (SheepFromHeaven) [inline src/roadmap-goals.md]: Typo: ChallEnges

File: src/roadmap-goals.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+
+*Goal:you can start right away and know where you are heading without being over- or underchallenged*
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Getting started
+
+The first part of your journey is to get an overview of the topics, that come across during your first weeks. You will learn which vision the onboarding process follows and get in touch with our toolset like JIRA and GIT/BITBUCKET
```
[the remark this answers is not in the document]
Comment (dannystey) [reply src/roadmap-goals.md]: removed jira and git/bitbucket

File: src/roadmap-goals.md [hunk 11]
```diff
@@ -0,0 +1,119 @@
+# Roadmap & Goals
+
+the roadmap gives you an overview and orientation for your first weeks at VI.
+
+## Goal
+> Every new frontend developer is able to implement components in our Biotope projects, in compliance with our VI standards!
+
+## Roadmap
+
+The following part is an overview of all the topics and challanges, that you are going to pass the next view weeks.
```
[the remark this answers is not in the document]
Comment (dannystey) [reply src/roadmap-goals.md]: fixed

File: src/roadmap-goals.md [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+### Getting started
+
+The first part of your journey is to get an overview of the topics, that come across during your first weeks. You will learn which vision the onboarding process follows and get in touch with our toolset like JIRA and GIT/BITBUCKET
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Fundamental I
+In that first part of the fundamentals of frontend development, you will be shown how to handle the mobile first approach, html semantics and css. Beside that you also get knowledge about accessability with WAI-ARIA and at least how to handle the SVG Icons.
```
[the remark this answers is not in the document]
Comment (dannystey) [reply src/roadmap-goals.md]: updated the text

File: src/roadmap-goals.md [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Testing
+In that topic you'll dive deeper into unit testing via diffrent tools and learn more about the TDD approach.
+
+__Challanges__
+- Build some unit tests with jest, mocha and chai
```
[the remark this answers is not in the document]
Comment (dannystey) [reply src/roadmap-goals.md]: therefore testing.md has to be updated as well

File: src/roadmap-goals.md (hunk 2 above)
Comment (dannystey) [reply src/roadmap-goals.md]: fixed

File: src/roadmap-goals.md (hunk 4 above)
Comment (dannystey) [reply src/roadmap-goals.md]: fixed

File: src/roadmap-goals.md (hunk 5 above)
Comment (dannystey) [reply src/roadmap-goals.md]: fixed

File: src/roadmap-goals.md (hunk 6 above)
Comment (dannystey) [reply src/roadmap-goals.md]: fixed

File: src/roadmap-goals.md (hunk 7 above)
Comment (dannystey) [reply src/roadmap-goals.md]: fixed

File: src/roadmap-goals.md (hunk 8 above)
Comment (dannystey) [reply src/roadmap-goals.md]: fixed

File: src/roadmap-goals.md (hunk 9 above)
Comment (dannystey) [reply src/roadmap-goals.md]: fixed

File: src/roadmap-goals.md (hunk 3 above)
Comment (dannystey) [reply src/roadmap-goals.md]: fixed

File: src/roadmap-goals.md [hunk 14]
```diff
@@ -0,0 +1,119 @@
...
+
+<roadmap-component>
+
+### Biotope - Basic Component
+Biotope is the VI default Frontend Ecosystem, which is implemented in every project of VI. You will be shown how biotope works and which architecture relies to it and how you can build simple components.
+
+__Challanges__
+- Biotope playground
+- Build a simple component
+- Templating with Handlebars
```
[the remark this answers is not in the document]
Comment (dannystey) [reply src/roadmap-goals.md]: updated it

**Comment by dannystey** [comment]
@jurekbarth could you review it again?

File: src/roadmap-goals.md [hunk 15]
```diff
@@ -0,0 +1,118 @@
+# Roadmap & Goals
+
+the roadmap gives you an overview and orientation for your first weeks at VI.
```
Comment (dweiger) [inline src/roadmap-goals.md]: Use uppercase T

File: src/roadmap-goals.md [hunk 16]
```diff
@@ -0,0 +1,118 @@
...
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Biotope - Basic Component
+Biotope is the VI default Frontend Ecosystem, which is implemented in every project of VI. You will be shown how biotope works and which architecture relies to it and how you can build simple components.
+
+__Challenges__
+- Biotope playground
```
Comment (dweiger) [inline src/roadmap-goals.md]: This point changed to "Explore Biotope Element"

File: src/roadmap-goals.md [hunk 17]
```diff
@@ -0,0 +1,118 @@
...
+- Paths
+- Biotope Resource Loader
+- Build an accordion
+
+*Goal: you understand how Biotope helps us to build scalable and reusable frontend frameworks and you are able to implement a simple component in a typical Biotope project*
+
+</roadmap-component>
+
+<roadmap-component>
+
```
Comment (dweiger) [inline src/roadmap-goals.md]: Is it on purpose that Security is missing?

**Pull request merged**